src/styles/blocks/fullCard.css

```diff
@@ -0,0 +1,8 @@
+.fullCard {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
width should be 200px
you can use  `box-sizing: border-box;` right?
so that it takes borders into consideration

Reply:
Yes, that's right. My bad ((